Assets/MixedRealityToolkit/Services/MixedRealityToolkit.cs

```diff
@@ -314,6 +315,53 @@ public void ResetConfiguration(MixedRealityToolkitConfigurationProfile profile)
 
         #endregion IMixedRealityServiceRegistrar implementation
 
+        private ExperienceScale scale;
+
+        /// <summary>
+        /// The scale (ex: World Scale) of the experience.
+        /// Updates the <see href="https://docs.unity3d.com/ScriptReference/XR.TrackingSpaceType.html">TrackingSpaceType</see> on the XR device.
+        public ExperienceScale AppScale
+        {
+            get => scale;
+            set
+            {
+                scale = value;
+
+                if (Application.isPlaying)
+                {
+                    TrackingSpaceType trackingSpace;
+
+                    // In current versions of Unity, there are two types of tracking spaces. For boundaries, if the scale
+                    // is not Room or Standing, it currently maps to TrackingSpaceType.Stationary.
+                    switch (scale)
+                    {
+                        case ExperienceScale.Standing:
+                        case ExperienceScale.Room:
+                            trackingSpace = TrackingSpaceType.RoomScale;
+                            break;
+
+                        case ExperienceScale.OrientationOnly:
+                        case ExperienceScale.Seated:
+                        case ExperienceScale.World:
+                            trackingSpace = TrackingSpaceType.Stationary;
+                            break;
+
+                        default:
+                            trackingSpace = TrackingSpaceType.Stationary;
+                            Debug.LogWarning("Unknown / unsupported ExperienceScale. Defaulting to Stationary tracking space.");
+                            break;
+                    }
+
+                    InputTracking.disablePositionalTracking = scale == ExperienceScale.OrientationOnly;
+
+                    if (!XRDevice.SetTrackingSpaceType(trackingSpace))
```

Comment:
>XRDevice.SetTrackingSpaceType

We should double check use of this API. It's deprecated in Unity 2019.3.
We were able to rework it by defining a new XR SDK based BoundarySystem. I'm not sure this logic should live in the core MRTK script, or at least not a direct API call.

https://github.com/microsoft/MixedRealityToolkit-Unity/blob/mrtk_development/Assets/MixedRealityToolkit.Providers/XRSDK/XRSDKBoundarySystem.cs#L84